keeper-gradle-plugin/src/main/java/com/slack/keeper/InferAndroidTestKeepRules.kt

```diff
@@ -47,13 +47,16 @@ import java.util.Locale
 public abstract class InferAndroidTestKeepRules : JavaExec() {
 
   @get:Classpath
-  public abstract val androidTestJar: RegularFileProperty
+  public abstract val androidTestSourceJar: RegularFileProperty
 
   @get:Classpath
-  public abstract val appJar: RegularFileProperty
+  public abstract val appTargetJar: RegularFileProperty
 
   @get:Classpath
-  public abstract val classpathJar: RegularFileProperty
+  public abstract val androidLib: RegularFileProperty
+
+  @get:Classpath
+  public abstract val androidTestLib: RegularFileProperty
```

Comment:
Can you document these? I don't know what "lib" means in this context

Reply:
Also, should these be marked as `@get:Optional` assuming they're only necessary for tracereferences? Don't want to wire these inputs if they're not using it if possible

Reply:
I guess this wording was from the plugin, but I decided to use the previous Jar suffix, which is probably more meaningful.
I also added the Optional annotation 👍